components/Templates/site_template.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="eng">
```

Comment:
I think the language code for English is  "en".

https://www.w3schools.com/tags/ref_language_codes.asp

Reply:
Thank you @cocarson, I will update it.